src/main/java/frc/robot/Buttons.java

```diff
@@ -1,30 +1,16 @@
 package frc.robot;
 
-public final class Buttons {
-    private Buttons() {}
-
-    // Buttons
-    public static final int A = 1;
-    public static final int B = 2;
-    public static final int X = 3;
-    public static final int Y = 4;
-    public static final int left_Bumper = 5;
-    public static final int right_Bumper = 6;
-    public static final int autoButton = 9; // will probably need to change
-    // Axes
-    public static final int leftJoystick_x = 0;
-    public static final int leftJoystick_y = 1;
-    public static final int leftTrigger = 2;
-    public static final int rightTrigger = 3;
-    public static final int rightJoystick_x = 4;
-    public static final int rightJoystick_y = 5;
-    // Dpad POVs
-    public static final int Dpad_up = 0;
-    public static final int Dpad_right = 90;
-    public static final int Dpad_down = 180;
-    public static final int Dpad_left = 270;
-
-    // auto buttons
-    public static final int autoShoot = A;
-    public static final int autoCollect = X;
```

Comment:
Could these methods be part of OperatorInterface?

Reply:
Yes